routes/comment.js

```diff
@@ -173,6 +174,72 @@ function createItemComment(req, res, next) {
     });
 }
 
+/**
+ * Update a comment
+ * @name update-comment
+ * @param {Object} params - The request URL parameters
+ * @param {string} params.project - The project ID
+ * @param {string} params.item - The item ID
+ * @param {string} params.comment - The comment ID
+ * @param {Object} body - The request body
+ * @param {string} body.body - Comment body
+ * @example
+ * curl -X PATCH -H "Content-Type: application/json" -d '{"body":"I updated this comment"}' https://host/v1/projects/00000000-0000-0000-0000-000000000000/items/77/comments/abcd-1234-abcd-1234
+ *
+ * {
+ *   "id": "abcd-1234-abcd-1234",
+ *   "createdBy": "userone",
+ *   "body": "I updated this comment",
+ *   "pin": {
+ *     "type": "Point",
+ *     "coordinates": [
+ *       0,
+ *       0
+ *     ]
+ *   },
+ *   "metadata": {},
+ *   "createdAt": "2017-10-23T17:13:25.585Z",
+ *   "updatedAt": "2017-10-23T17:13:25.585Z"
+ * }
+ */
+function updateItemComment(req, res, next) {
+  const commentId = req.params.comment;
+  const validBodyAttrs = ['body'];
```

Comment:
We should also allow the ability to update the coordinates.

Reply:
See the outdated discussion above in server.js about this.